src/send/ValidateRecipientAccount.tsx

```diff
@@ -263,11 +263,13 @@ export class ValidateRecipientAccount extends React.Component<Props, State> {
   }
 
   render = () => {
-    const { t, recipient, error } = this.props
-    const { singleDigitInputValueArr } = this.state
+    const { t, recipient, error, addressValidationType } = this.props
+    const { singleDigitInputValueArr, inputValue } = this.state
     const displayName = getDisplayName(recipient, t)
     const isFilled =
-      singleDigitInputValueArr.filter((entry) => /[a-f0-9]/gi.test(entry)).length === 4
+      addressValidationType === AddressValidationType.FULL
```

Comment:
the full address validation flow was broken, since that flow doesn't touch `singleDigitInputValueArr`, this `isFilled` was always false and i could not press the submit button

Reply:
oh good catch! 👍 
Should we cover that with a test? 😅